plugins/services/src/js/components/DeploymentsModal.js

```diff
@@ -229,7 +229,7 @@ class DeploymentsModal extends mixin(StoreMixin) {
         href={`#/services/overview/${encodeURIComponent(servicePath)}`}
         onClick={this.props.onClose}
       >
-        {`services${servicePath}`}
+        {`services${ServicePath}`}
```

Comment:
Hi @SuperOmario, thanks for the contribution! I bet it's the first `services` that has to be changed.